### PR TITLE
Friction coefficients are added. The closer to zero, the more friction affects.

### PR DESCRIPTION
**Pull request by binitshah**
Moungsung's PR for adding friction to the field.

### CODE REVIEW

File: data/field/field.urdf [hunk 1]
```diff
@@ -9,6 +15,13 @@
             <geometry>
                 <mesh filename="field.obj" scale="1 1 1"/>
             </geometry>
+   
+<friction>      
+        <ode>
```
Comment (binitshah) [inline data/field/field.urdf:20]: I've been unable to find where these `<friction>` and `<ode>` tags come from. Could you link me a reference.

File: data/field/field.urdf [hunk 2]
```diff
@@ -1,6 +1,12 @@
 <?xml version="1.0" ?>
 <robot name="field.urdf">
     <link name="field_link">
+    <contact>
```
Comment (binitshah) [inline data/field/field.urdf:4]: Could you post a link for where you found documentation for these tags?

File: data/field/field.urdf (hunk 1 above)
Comment (msunglim) [reply data/field/field.urdf:20]: I tried the <friction> and <ode> to the Field.urdf but it didn't work so I found the friction from a Japanese website. Here is the link :https://qiita.com/saketosakana/items/0e52997cbe43919ee2bd 
https://qiita.com/saketosakana/items/491d3e0cec0a0e288748
https://qiita.com/saketosakana/items/b4529df7ebc1f60356a6

File: data/field/field.urdf (hunk 2 above)
Comment (msunglim) [reply data/field/field.urdf:4]: I tried the <friction> and <ode> to the Field.urdf but it didn't work so I found the friction from a Japanese website. Here is the link :https://qiita.com/saketosakana/items/0e52997cbe43919ee2bd 
https://qiita.com/saketosakana/items/491d3e0cec0a0e288748
https://qiita.com/saketosakana/items/b4529df7ebc1f60356a6

**Pull request closed without merging**